### DISCUSSION TITLE
Filecoin v16 Skyr 升级 - 社区 AMA 常见问题解答

### DISCUSSION THREAD

**Discussion by kaitlin-beegle** [filecoin-project/community-china#215] [社区公告]
**问：哪个版本的 Lotus 可运行 FVM （Filecoin 虚拟机）？**
答：两种Lotus 版本 v1.16.0 和 v1.17.0 都将支持FVM。这两个版本目前都是预发布版本；完整版本将在 7 月 6 日升级之前发布。

**问：我什么时候需要升级到最新版本的 Lotus？**
答：您需要于epoch 1960320 前升级您的节点, 时间大约是 2022 年 7 月 6 日14:00 UTC。有关官方网络升级通知，请参阅[此处](https://status.filecoin.io/incidents/9l5jq51snhnb)。有关社区网络讨论，请参阅[此处](https://github.com/filecoin-project/community/discussions/74?sort=new#discussioncomment-2965298)。

**问：FVM 会影响燃料费用或网络吞吐量吗？**
答：也是也不是。随着 Filecoin 虚拟机 引入 WASM 字节码，网络现在可以对执行智能合约(actor)逻辑的实际成本进行定价。为支持这应用，[FIP0032](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0032.md) 引入了三种新的燃料：执行燃料、系统调用燃料和外部燃料。计算燃料成本的网络基础费用将保持不变。但为了确保在引入 FVM 期间基本费用不会飙升，FVM 工程团队对新的燃料参数进行了广泛的测试（参见：[这儿](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0032.md#incentive-considerations)和[这里](https://github.com/raulk/fil-fip-0032-data/)）。 CrytpoEconLab 还进[行了分析](https://github.com/filecoin-project/FIPs/discussions/316#discussioncomment-2758676)，以确保网络的持续稳定性。

**问：我们正在做哪些工作来支援 FVM <> EVM 相容性？**
答：许多社区成员表示 EVM 相容性是重要的优先事项。因此，现在EVM 相容性将在 2022 年秋季作为 FVM 里程碑 M2.1 发布。这项工作将在社区合作伙伴的支持下逐步部署在测试网中。 FVM 团队已经完成了大量的研究，并实现一个可用的 EVM 字节码执行环境！我们在 [FIL Toronto](https://fil-toronto.com/) 将提供高层级演示。

**问：EVM actor 是否能够将方法调用原生 FVM 的内置 actor？哪些功能不适用于 EVM actor，而 FVM actor可以使用？**
答：EVM actors 可以调用 FVM actors，反之亦然。所有actors都将以相同的方式互动，并且已经备好额外的Solidity 库来减少actor之间的冲突。其功能上的主要区别在于，FVM 能够提供actor“升级”功能，并且能够更有效的使用现有的 IPLD 功能。例如，FVM 将能廉价地将其状态树的不可变副本发送给另一个actor, 这部分EVM 则无法做到。虽然其中一些功能可以通过预编提供给 EVM actor，但它们的实用性和局限需要依具体情况进一步研究。

**问：在将合约从 EVM 区块链分叉到 FVM actor时，是否需要考虑任何新的安全漏洞？**
答：不需要。为了防止出现安全漏洞，设有一个‘双重’沙盒：EVM 沙盒和 FVM/WASM 沙盒。但是，重要的是要记住错误可能发生在任何层级。我们还需要注意 EVM 和 FVM/WASM 之间可能产生灰色区域的潜在“不和谐”或不匹配。

**Comment by RichByDMC** [comment]
挖FIL的伙伴们看过来，一个火爆全网的全新存储项目自带公链钱包NFT等完整生态DMC，唯一能够超过Fil的项目诞生了。主要还能双挖FIL和DMC~~有想法的可以V我：dmctorich
